lib/bn.js

```diff
@@ -715,13 +715,214 @@ BN.prototype._bigMulTo = function _bigMulTo(num, out) {
 
 BN.prototype.mulTo = function mulTo(num, out) {
   var res;
-  if (this.length + num.length < 63)
+  var len = this.length + num.length;
+  if (len < 63)
     res = this._smallMulTo(num, out);
-  else
+  else if (len < 1024)
     res = this._bigMulTo(num, out);
+  else
+    res = this._bigMulToF(num, out);
   return res;
 };
 
+
+// Private
+
+var FFTM = function (x, y) {
```

Comment:
This clearly doesn't fit into the rest of the file. Please use `function FFTM(...)`. Also please avoid using anonymous functions.